CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 - No changes yet.
 
+## [1.21.0] - 2019-07-23
```

Comment:
The new entries will go under the Unreleased section above.

(Also, seeing as it's a bugfix, when it gets cut, it'll be a 1.20.1
unless there's also a feature in here somewhere.)

Reply:
> The new entries will go under the Unreleased section above

Thanks! I updated it. 👍

> when it gets cut, it'll be a 1.20.1

What can I expect in terms of timeline?